data/2021_pyladies_olomouc_autumn/tasks/5_str.yaml

```diff
@@ -0,0 +1,98 @@
+tasks:
+
+- section:
+    markdown: |
+      Procvičení řetězců. Mějte při ruce taháky!
+
+- id: handout5_00
+  markdown: |
+    Napište program, který se zeptá na dva řetězce a zjistí,
+    jestli je jeden obsažen ve druhém.
+    Nebere přitom ohled na velikost písmen.
+
+- id: handout5_01
+  markdown: |
+    Napište program, který se zeptá na příjmení uživatelky/uživatele
+    a *zkusí* podle něj uhodnout její/jeho pohlaví.
+    To pak vypíše.
+
+    (Pohlaví není podle příjmení možné určit přesně – stačí zvládnout ta
+    nejčastější česká příjmení.)
```

Comment:
Tohle hádání pohlaví dle příjmení bych dal úplně pryč, místo toho např. program, který bude umět přepsat světovou stranu do zkratky:
- input: severovýchod
- output: SV

Plus pokud tohle zvládnou, takže ještě další variantu této úlohy, která bude umět i přesnější určení strany, např.:
severoseverovýchod -> SSV
západojihozápad -> ZJZ

(Nebo ještě přidat i obrácenou variantu, ze zkratek dělat název strany.)

Reply:
Přidáno.